packages/react-percy-webpack/src/entry/index.js

```diff
@@ -1,27 +1,27 @@
+import { EntryNames } from './constants';
 import findSnapshotFiles from './findSnapshotFiles';
+import getEntryPath from './getEntryPath';
 import merge from 'webpack-merge';
-import path from 'path';
+import writeRenderEntry from './writeRenderEntry';
+import writeSnapshotsEntry from './writeSnapshotsEntry';
 
 export default function configureEntry(webpackConfig, percyConfig) {
-  const snapshotFiles = findSnapshotFiles(percyConfig);
+  const frameworkEntryFile = getEntryPath(EntryNames.framework);
 
-  const entry = snapshotFiles.reduce((prev, file) => {
-    const entryName = path
-      .relative(percyConfig.rootDir, file)
-      .replace(new RegExp(`${path.extname(file)}$`), '');
-    return {
-      ...prev,
-      [entryName]: file,
-    };
-  }, {});
+  const renderEntryFile = getEntryPath(EntryNames.render);
+  writeRenderEntry(percyConfig, renderEntryFile);
 
-  if (percyConfig.includeFiles && percyConfig.includeFiles.length) {
-    entry['__percy_include__'] = percyConfig.includeFiles;
-  }
+  const snapshotFiles = findSnapshotFiles(percyConfig);
+  const snapshotsEntryFile = getEntryPath(EntryNames.snapshots);
+  writeSnapshotsEntry(snapshotFiles, snapshotsEntryFile);
```

Comment:
The other big change I made here was generating a single JS file on disk containing all the snapshot files. Each snapshot file is still automatically wrapped in a suite, but having them all in this single file makes life a bit easier for us elsewhere in the code and should make webpack compiling faster on larger projects than the previous configuration.